e2e/cypress/integration/literatureRelations.test.js

```diff
@@ -95,7 +95,7 @@ describe('Assign Conference', () => {
       );
       cy.get('[data-test-id="assign-conference"]').click();
       cy.get('.ant-drawer-content', { timeout: 5000 }).should('be.visible');
-      cy.get('.ant-drawer-content').find('[type="button"]').first().click();
+      cy.get('.ant-drawer-content').find('[type="button"]').eq(1).click();
```

Comment:
isn't first index 0?

Reply:
Before the close drawer button didn't have type='button'. After update, the .ant-drawer-content has now three ```[type="button"]``` elements - the first one is the close button, second is the search button that we aim for here, and the last one being assign button. Hence the necessary change